### ISSUE TITLE
Send a message (to any destiny)

### ISSUE THREAD

**Issue by rodrigomonelos** [saycel/webph.one#125]
- [ ] See the conversation updated with the sent message.
- [ ] Scroll up to the end of the conversation
- [ ] Keep the keyboard open

**Comment by gmarcos87** [comment]
How is this different from what is already there? What does it mean to any destination?
I also do not think I can force the keyboard to stay open.

**Comment by rodrigomonelos** [comment]
I agree with you that this was almost already working, but we didn't have any backlog item defining this behaviour.

- As we talked over Telegram, sometimes messages are not going out, nor being showed. Sometimes they are showed after a while. The first acceptance criteria of this issue defines this behaviour. I added "just after tapping on the send button" to consider the delay.

- I added an acceptance criteria about seeing the message on the receiver. It could be considered part of #115 also, or we can create a new issue. Let me know if you prefer to have this acceptance criteria somewhere else.

- I added "to any destiny" on the title to differenciate this backlog item from #111, that is focused on reaching GSM phones and not in the behaviour.

- I think that keeping the keyboard open is valuable. Can you confirm if doing this is a PWA limitation (we sould add it to our list)? If it is not a limitation but is complex, please share that info so we decide toghether if we invest that effort on it now.
